lib/tasks/dev.rake

```diff
@@ -0,0 +1,89 @@
+desc "Fill the database tables with some sample data"
+task({ :sample_data => :environment}) do
+  
+  User.destroy_all
+  Loan.destroy_all
+  Adjustment.destroy_all
+
+  # Create Sample Users
+    #  email                  :string           default(""), not null
+    #  encrypted_password     :string           default(""), not null
+  email = ["alice@example.com", "bob@example.com", "carol@example.com"]
+
+  3.times do |count|
+    user = User.new
+    user.email = email.at(count)
+    user.password = "password"
+    user.save
+  end
+
+  p "Added #{User.count} Users"
+
+  # Create Sample Loans
+    #  current_balance     :float
+    #  original_amount     :float
+    #  interest_rate       :float
+    #  periods_in_year     :integer
+    #  user_id             :integer
+    #  loan_name           :string
+    #  monthly_min_payment :float
+
+    #cur_bal = [1000, 5000, 10000, 15000, 20000, 25000, 30000, 35000, 40000]
+    org_bal = [1000, 5000, 10000, 15000, 20000, 25000, 30000, 35000, 40000]
+    int_rate = [2, 3, 4, 5, 6]
+    payback_per = [2, 4, 6]
+    per_in_year = 12
+    users = User.all
+    l_name = ["Car", "Car 2", "Student Loan", "Grad School", "Credit Card"]
```

Comment:
How about `loan_name` here (and similarly above, `periods_in_year` rather than `per_in_year`, etc)? Do the 3 saved characters really matter, given autocomplete etc in our editors these days; versus the readability benefits later on in this file for teammates who jump in?

Reply:
I think I was worried about naming the array the same as the attribute I was trying to assign it too.  I agree it will be more readable to bring that in and pluralize it.  I will make these adjustments.

Reply:
Ah, that makes sense. Another thing people sometimes do is to use constants instead of variables to communicate "magic numbers":

https://eliotsykes.com/2015/08/18/magic-numbers/

Reply:
Thanks, used the `magic numbers` technique such as `EXAMPLE_LOAN_NAMES` so I could be more explicit.  It should make things more reviewable later on.